### PR TITLE
Adding precession package to conda-forge

### PR DESCRIPTION
**Pull request by chadhenshaw**

### CODE REVIEW

**Comment by chadhenshaw** [comment]
@conda-forge-admin, please ping @conda-forge/help-python, ready for review

**Comment by BastianZim** [comment]
Hi @chadhenshaw thanks for your contribution. Please redo your recipe with Grayskull again and follow the checklist that was originally in the PR body before you deleted it. Additionally, all comments of the linter should be solved.
Once that is done, you can request a review again.

**Comment by ndevenish** [comment]
Also, please follow the checklist, as requested. It can be found [here](https://github.com/conda-forge/staged-recipes/blob/0df3f30ec0ab43e1db3919c7e84fe192deb8fe47/.github/pull_request_template.md).

Specifically, the PR title should reflect the package, not "Create meta.yaml", as this makes it harder to see the history what has been merged in.

File: recipes/precession/meta.yaml
Comment (ndevenish) [inline recipes/precession/meta.yaml:31]:
```suggestion
test:
  imports:
    - precession
    - precession.test
  commands:
    - pip check
  requires:
    - pip
```

Comment (ndevenish) [reply recipes/precession/meta.yaml:31]: The `pip check` stage that Grayskull adds is usually a useful cross-check that the dependency tree checks out.

**Comment by chadhenshaw** [comment]
@conda-forge-admin, please ping @conda-forge/help-python, I've updated as requested, can you please review?

**Comment by chadhenshaw** [comment]
@ndevenish Is there anything else that needs to be done before this can be merged? I believe I have completed all the checklist items.

**Comment by ndevenish** [comment]
I think you've fulfilled the checklist now, you just need to wait for review.

**Review by carterbox — APPROVED** [review]

**Pull request merged**
